x11-misc/j4-dmenu-desktop/j4-dmenu-desktop-2.17.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+MY_PV="r${PV}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="A fast desktop replacement for i3-dmenu-desktop"
+HOMEPAGE="https://github.com/enkore/j4-dmenu-desktop"
+SRC_URI="https://github.com/enkore/j4-dmenu-desktop/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
```

Comment:
Order following the example set by `skel.ebuild` would be
```
LICENSE
SLOT
KEYWORDS
IUSE
```

Reply:
Strange, you are right, don't know, why my vim did it wrong. Will look into it, but changed now. Thanks!